internal/models/schema-recipe.go

```diff
@@ -869,35 +871,19 @@ func (n *NutritionSchema) UnmarshalJSON(data []byte) error {
 
 	switch x := v.(type) {
 	case map[string]any:
-		if val, ok := x["calories"].(string); ok {
-			n.Calories = strings.TrimSpace(val)
-		}
-
-		if val, ok := x["carbohydrateContent"].(string); ok {
-			n.Carbohydrates = val
-		}
-
-		if val, ok := x["cholesterolContent"].(string); ok {
-			n.Cholesterol = val
-		}
-
-		if val, ok := x["fatContent"].(string); ok {
-			n.Fat = val
-		}
-
-		if val, ok := x["fiberContent"].(string); ok {
-			n.Fiber = val
-		}
-
-		if val, ok := x["proteinContent"].(string); ok {
-			n.Protein = val
-		}
-
-		if val, ok := x["saturatedFatContent"].(string); ok {
-			n.SaturatedFat = val
-		}
-
-		if val, ok := x["servingSize"].(string); ok {
+		n.Calories = regex.Digit.FindString(strings.Replace(extensions.ConvertToString(x["calories"]), ",", ".", -1))
```

Comment:
As the -1 argument suggests there is no limit on the number of replacements, we can instead use strings.ReplaceAll. For example:
```go
n.Calories = regex.Digit.FindString(strings.ReplaceAll(extensions.ConvertToString(x["calories"]), ",", "."))
```

Reply:
Most definitely, thanks. 👍